models.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"time"
+)
+
+// request and response structures
+
+// Response structure for /status call.
+type StatusResponse struct {
+	Task_id   string     `json:"task_id"`
```

Comment:
nit: Consistent naming, `TaskID` instead of `Task_id`.

Reply:
just installed a plugin to that suggested this, implemented.